sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -720,8 +720,8 @@ case class DescribeColumnCommand(
     }
 
     val catalogTable = catalog.getTempViewOrPermanentTableMetadata(table)
-    val colStats = catalogTable.stats.map(_.colStats).getOrElse(Map.empty)
-    val cs = colStats.get(field.name)
+    val colStats = catalogTable.stats.map(_.colStats.map{ case (key, value) => key.toLowerCase -> value}).getOrElse(Map.empty)
+    val cs = colStats.get(field.name.toLowerCase())
```

Comment:
Can you link this behaivour to `SQLConf.caseSensitiveAnalysis`?

Reply:
@maropu @dongjoon-hyun  i have done the changes. could you  kindly review?